docker-for-windows/index.md

```diff
@@ -48,148 +45,78 @@ and make sure `docker` commands are working properly.
+    Download complete
+    ca4f61b1923c:
+    Pull complete
+    Digest: sha256:97ce6fa4b6cdc0790cda65fe7290b74cfebd9fa0c9b8c38e979330d547d22ce1
+    Status: Downloaded newer image for hello-world:latest
+
+    Hello from Docker!
     This message shows that your installation appears to be working correctly.
-
-    To generate this message, Docker took the following steps:
-    1. The Docker client contacted the Docker daemon.
-    2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
-    3. The Docker daemon created a new container from that image which runs the executable that produces the output you are currently reading.
-    4. The Docker daemon streamed that output to the Docker client, which sent it to your terminal.
+    ...
     ```
-
-4.  Try something more ambitious, and run an Ubuntu container with this command.
+3.  List the hello-world image:
 
     ```none
-    PS C:\Users\jdoe> docker run -it ubuntu bash
+    docker image ls
     ```
 
-    This downloads the `ubuntu` container image and start it. Here is the output of running this command in a powershell.
+4.  Run an image of the Ubuntu OS:
 
     ```none
-    PS C:\Users\jdoe> docker run -it ubuntu bash
-
-    Unable to find image 'ubuntu:latest' locally
-    latest: Pulling from library/ubuntu
-    5a132a7e7af1: Pull complete
-    fd2731e4c50c: Pull complete
-    28a2f68d1120: Pull complete
-    a3ed95caeb02: Pull complete
-    Digest: sha256:4e85ebe01d056b43955250bbac22bdb8734271122e3c78d21e55ee235fc6802d
+    PS C:\Users\Gordon> docker run -it ubuntu bash
+
+    docker : Unable to find image 'ubuntu:latest' locally
+    ...
+
+    latest:
+    Pulling from library/ubuntu
+    22dc81ace0ea:
+    Pulling fs layer
+    1a8b3c87dba3:
+    Pulling fs layer
+    91390a1c435a:
+    Pulling fs layer
+    ...
+    Digest: sha256:e348fbbea0e0a0e73ab0370de151e7800684445c509d46195aef73e090a49bd6
     Status: Downloaded newer image for ubuntu:latest
     ```
 
-    Type `exit` to stop the container and close the powershell.
+5.  Open a second PowerShell tab and list all the containers.
 
-5.  Start a Dockerized webserver with this command:
+    The hello-world container exited after displaying its message. The Ubuntu container is still running:
 
     ```none
-    PS C:\Users\jdoe> docker run -d -p 80:80 --name webserver nginx
+    PS C:\Users\Gordon> docker container ls --all
+    CONTAINER ID      IMAGE            COMMAND       CREATED            STATUS                        PORTS      NAMES
+    64b5205999ed      ubuntu           "bash"        2 minutes ago      Up 2 minutes                             laughing_kowalevski
+    45f77eb48e78      hello-world      "/hello"      3 minutes ago      Exited (0) 3 minutes ago                 relaxed_sammet
     ```
 
-    This downloads the `nginx` container image and start it. Here is the
-    output of running this command in a powershell.
+6.  In the first PowerShell tab, press `Ctrl+c` to stop the Ubuntu conatiner.
```

Comment:
That really works? (Ctrl-c)?  I would recommend typing `exit` instead.

And container, not conatiner.  We should have spell-checking phases in the checks.

Reply:
@akimd , my terminal does not return the prompt ... not sure why not.

Reply:
It seems that you cannot run an interactive session with PowerShell? The "Command Prompt" terminal works. But you cannot stop or remove all containers in Command Prompt (but PowerShell works). 

How do I run an interactive session with PowerShell? Do I need to do something different? 

cc: @akimd @thaJeztah

Reply:
Solved. I was running PowerShell ISE, not PowerShell: https://github.com/docker/for-win/issues/223